docs/building-apps.md

```diff
@@ -249,3 +254,23 @@ Similar to the [`enact serve`](./serving-apps.md) command, the watcher will buil
 
 ## Stats Analysis
 The Bundle analysis file option uses the popular [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) to create a visual representation of the project build to **stats.html**, showing the full module hierarchy arranged by output size. This can be very useful in determining where bloat is coming from or finding dependencies that may have been included by mistake.
+
+## Override metadata
+The @enact/cli tool will check the project’s package.json looking for an optional `enact` object for a few [customization options](https://enactjs.com/docs/developer-tools/cli/starting-a-new-app/#enact-project-settings).
+For supporting additional options, Enact CLI provides `--meta` option to add a JSON to override package.json enact metadata.
+
+For example:
+```bash
+enact pack --meta='{"title":"myapp"}'
+```
+
+This example is the same as package.json below.
+```js
+{
+	...
+	"enact": {
+		"title": "myapp"
+	}
+	...
+}
```

Comment:
This means we don't need to describe the metadata in original json file again in new json file, is it right?
If it's right, I think this example is OK.

Reply:
Yes, through this example, the new json(`enact pack --meta`) format will be overridden without having to modify the original json(`package.json`).